runtime/dolphin/src/lib.rs

```diff
@@ -125,7 +125,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
     spec_name: create_runtime_str!("dolphin"),
     impl_name: create_runtime_str!("dolphin"),
     authoring_version: 2,
-    spec_version: 4060,
+    spec_version: 4061,
```

Comment:
not sure we need to keep these in sync anymore

Reply:
Yeah doesn't really matter anymore but I would still think keeping them synced makes the most sense